osCam/core/models.py

```diff
@@ -3,4 +3,4 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.utils import timezone
-from django.utils.translation import activate
+from django.utils.translation import activate
```

Comment:
I see you've added a new import but don't see anything in the file that is using it. What was the purpose?